src/main/java/edu/slu/auxiliary/CompressorService.java

```diff
@@ -0,0 +1,142 @@
+/*
+ * Gimme some JSON.  I'll give you back less JSON.
+ * @author bhaberbe
+
+ * @notes
+ * In my mind, someone is calling this hoping RERUM knows how to compress it.
+ * If they were to go as far as providing the keys to be removed, they would just do it themselves.
+ * Instead, you provide the Compressor keys to keep, and let it go to work on your JSON objects.  
+ * We can have default keys we know we must keep.
+ * We can have default keys we know must bee removed.
+ * Anything else has to be provided.  Keys to keep can be provided.  
+ */
+package edu.slu.auxiliary;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Set;
+import net.sf.json.JSONObject;
+
+public class CompressorService{
+    
+    //These are keys that we want to make sure stay with the JSONObject.
+    //Think about things like "target" and "on"...are they special?  Current thinking is no, they are just as viable to be removed as any other property. 
+    
+    private Set<String> primitiveKeys = new HashSet<>(Arrays.asList("@id",
+        "id", 
+        "type", 
+        "@type", 
+        "label", 
+        "name", 
+        "summary", 
+        "description", 
+        "@context"
```

Comment:
maybe `title` `value` `@value` and the itemList things like `items` `ItemListElement` `members`? I'm not sure what fits best here.